manifests/manifests.go

```diff
@@ -25,6 +25,7 @@ import (
 // FS embeds the manifests
 //
 //go:embed profiles/* profiles/prom/* profiles/prom-thanos/* profiles/thanos/* profiles/infra/plugins/* profiles/infra/capitpls/*
+//go:embed profiles/fleet/* profiles/fleet/plugins/*
```

Comment:
Is there a condition compile way to remove unnecessary profile for fleet manager

Reply:
will do it later in a seprated PR